.github/workflows/cont_integration.yml

```
@@ -1,7 +1,6 @@
+name: bdk-ffi CI
 on: [push, pull_request]
```

As above, but only need to run when bdk-ffi files change.
```suggestion
on:
    push:
             paths:
                - 'bdk-ffi/**'
          pull_request
             paths:
                - 'bdk-ffi/**'
```

---

Oh those are fantastic changes, I figured I'd look into it after but it's too easy not to include!